javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,37 @@
+const width = 5;
+const height = 10;
+console.log('Width: ', width);
+console.log('height: ', height);
+const area = width * height;
+console.log('area: ', area);
+console.log('area has the type of ', typeof area);
+const payment = 10;
+const bill = 5;
+const change = payment - bill;
+console.log('payment ', payment);
+console.log('bill ', bill);
+console.log('change: ', change);
+console.log('change has the type of ', typeof change);
+const quizzes = 100;
+const midterm = 99;
+const final = 98;
+const grade = (quizzes + midterm + final) / 3;
+console.log('grade: ', grade);
+console.log('grade has the type of ', typeof change);
+const firstName = 'Tyler';
+const lastName = 'Stahl';
+const fullName = firstName + ' ' + lastName;
+console.log('fullName: ', fullName);
+console.log('fullName has the type of ', typeof fullName);
+const pH = 13;
+const isAcidic = pH < 7;
+console.log('isAcidic: ', isAcidic);
+console.log('isAcidic has the type of ', typeof isAcidic);
+const headCount = 300;
+const isSparta = headCount === 300;
+console.log('isSparta: ', isSparta);
+console.log('isSparta has the type of ', typeof isSparta);
+let motto = fullName;
+motto = fullName + ' ' + 'is the GOAT';
```

Comment:
Rather than reassigning the whole thing again, you should be utilizing the `+=` operator, so you can keep the old value, and add to it